package.json

```diff
@@ -86,52 +86,52 @@
     "moment": "~2.29.4",
     "moment-range": "^4.0.2",
     "qs": "^6.10.3",
-    "react": "18.0.0",
+    "react": "18.1.0",
     "react-apollo": "~3.1.5",
-    "react-dom": "18.0.0",
+    "react-dom": "18.1.0",
     "react-hook-form": "^6.11.0",
-    "react-native": "0.69.6",
+    "react-native": "0.70.5",
     "react-native-app-intro-slider": "^4.0.4",
     "react-native-autocomplete-input": "^5.1.0",
     "react-native-calendars": "1.1276.0",
     "react-native-collapsible": "^1.6.0",
     "react-native-elements": "^3.4.3",
     "react-native-expo-image-cache": "4.1.0",
-    "react-native-gesture-handler": "~2.5.0",
+    "react-native-gesture-handler": "~2.8.0",
     "react-native-get-random-values": "~1.8.0",
     "react-native-gifted-chat": "0.16.3",
-    "react-native-maps": "0.31.1",
+    "react-native-maps": "1.3.2",
```

Comment:
whuu big step. did you check that in detail?

Reply:
yes it's a big update but when I read the changes I don't think we have to make any changes for us.
[releases](https://github.com/react-native-maps/react-native-maps/releases)

Reply:
all right